lgsm/config-default/config-lgsm/mcserver/_default.cfg

```diff
@@ -12,10 +12,8 @@
 javaram="1024" # -Xmx$1024M
 
 ## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
-fn_parms(){
```

Comment:
That cannot work without the function here.
This is unfortunatly a hard dependency to my fix

Reply:
Still breaks minecraft currently (2 executeable vars + eval for executeable would be needed)